### PR TITLE
#45 - configure php debugging environment

### LINKED ISSUE #45: Configure PHP Debugging Environment
As in title.

### PR DESCRIPTION
**Pull request by plencka**
It should close #45.

### CODE REVIEW

File: environment/php/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+RUN if [ ${XDEBUG_INSTALL} = true ]; then \
+    apk --no-cache add $PHPIZE_DEPS \
+    && pecl install xdebug-${XDEBUG_VERSION}\
+    && docker-php-ext-enable xdebug \
+    && echo "xdebug.client_host=${XDEBUG_HOST}" >> /usr/local/etc/php/conf.d/xdebug.ini \
+    && echo "xdebug.client_port=${XDEBUG_PORT}" >> /usr/local/etc/php/conf.d/xdebug.ini \
+    && echo "xdebug.mode=${XDEBUG_MODE}" >> /usr/local/etc/php/conf.d/xdebug.ini \
+    && echo "xdebug.start_with_request=${XDEBUG_START_WITH_REQUEST}" >> /usr/local/etc/php/conf.d/xdebug.ini \
+    && echo "xdebug.log_level=${XDEBUG_LOG_LEVEL}" >> /usr/local/etc/php/conf.d/docker-php-ext-xdebug.ini \
+;fi
```
Comment (krzysztofrewak) [inline environment/php/Dockerfile:21]: Please add a blank line at the EOF.

**Review by krzysztofrewak — CHANGES_REQUESTED** [review]

File: environment/php/Dockerfile (hunk 1 above)
Comment (plencka) [reply environment/php/Dockerfile:21]: Oh no, my weakness! I'm on it.

File: environment/php/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,21 @@
+ARG VERSION=8.0.7.0
+FROM ghcr.io/blumilksoftware/php:$VERSION
+
+ARG XDEBUG_VERSION=3.0.4
+ARG XDEBUG_HOST=172.17.0.1
+ARG XDEBUG_PORT=9021
```
Comment (mtracz) [inline environment/php/Dockerfile:6]:
```suggestion
ARG XDEBUG_PORT=9003
```

File: docker-compose.yml [hunk 3]
```diff
@@ -14,7 +14,16 @@ services:
       - "${EXTERNAL_WEBSERVER_PORT}:80"
 
   php:
-    image: ghcr.io/blumilksoftware/php:8.0.7.0
+    build:
+      context: ./environment/php
+      dockerfile: Dockerfile
```
Comment (mtracz) [inline docker-compose.yml:19]:
```suggestion
```

**Review by mtracz — CHANGES_REQUESTED** [review]

**Review by krzysztofrewak — APPROVED** [review]

**Review by mtracz — APPROVED** [review]

**Review by LukaszPodlipski — APPROVED** [review]

**Pull request merged**